samples/TagHelperSample.Web/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
     "commands": {
-        "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
+        "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001/taghelpers",
```

Comment:
@davidfowl this is where the "Started" message really sucks, can we get the url printed out?